apps/directory/src/stores/biobanksStore.js

```diff
@@ -120,10 +120,10 @@ export const useBiobanksStore = defineStore("biobanksStore", () => {
         let biobanksWithCollections = biobankResult.Biobanks;
         if (filtersStore.hasActiveFilters) {
           biobanksWithCollections = biobanksWithCollections.filter(
-            (biobank) => biobank.collections
+            (biobank) => biobank.collections?.length
           );
         }
-        biobankCards.value = filterWithdrawn(biobankResult.Biobanks);
```

Comment:
I think this fixes #3577

Reply:
This was indeed something that was adjusted in the race condition PR: #3562